### DISCUSSION TITLE
Is there discord server

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [microsoft/onnxruntime#11681] [Other Q&A] [answered]
Does anyone know if there is a discord server for this repository?

**Comment by Craigacp** [comment, marked as answer]
There's a [Slack server](https://lfaifoundation.slack.com/) for LF AI & Data which includes the ONNX project and a bunch of the ONNX converters, but ONNX Runtime is discussed here on Github.

**Comment by GeorgeS2019** [comment]
@Craigacp  It seems the slack server is only open to paid members

**Reply by Craigacp** [reply]
I don't believe it is. It was migrated from a freely available gitter instance some years ago, and my employer isn't a paid member of the LF AI & Data group yet I still have access. You might need to use the signup link mentioned here - https://github.com/onnx/onnx/blob/main/README.md#discuss.
